README.md

```diff
@@ -54,11 +54,13 @@ After setting up an account with us we will send you onboarding information. It
 
 [OnboardID iOS SDK](./onboard_id_ios.md)
 
+[OnboardID React Native SDK](./onboard_id_theme_reactnative.md)
+
```

Comment:
Rather the correct link for this would be

https://www.npmjs.com/package/@netki/netki-mobilesdk?activeTab=readme